modules/nf-core/ashlar/main.nf

```diff
@@ -0,0 +1,40 @@
+process ASHLAR {
+    tag '$meta.id'
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::ashlar=1.17.0" : null)
+    /* TODO nf-core: we'll provide a dockerfile as described in 5) under Software Requirements
```

Comment:
Since Ashlar (and most of our own modules) are written in Python, it actually wouldn't be a big lift to make them available via bioconda. I guess we need to consider whether having duplicate containers aside from our own "official" ones is OK, or whether we just want to delegate to bioconda as our official containers anyway.

Reply:
Yeah, adding to bioconda would streamline this process, but as I mention in the question above, there's supposed to be a way for us to provide a Dockerfile that they then build and maintain a container for.  Hopefully they'll respond to my question about this soon.

Reply:
I still haven't gotten any reply to my question about this on the nf-core Slack modules channel.  I went ahead and tried using our Docker Hub image and that seems to work fine.  It passed both the lint and functional tests.  I've posted another question on Slack asking if this is acceptable or if they'd prefer we provide a Dockerfile.

Reply:
I got a quick response this time:

> So, nf-core modules usually use the Biocontainers that are automatically produced by the Bioconda tooling. That helps us be sure of consistency, and you also get the singularity container as a bonus (if you look at a couple of modules you’ll see what I mean).
Yours should be present at https://quay.io/repository/biocontainers/ashlar. I can’t see it, which probably means it’s not been made public (which unfortunately doesn’t happen by default). If you go to https://gitter.im/biocontainers/Lobby and ask nicely, someone (probably Bjoern) will make it public for you, and you can use it (and the singularity container) here. You also won’t have to maintain your own version of the container (unless you have other reasons for doing so).

So it does look like ashlar is there.  I found it at:
https://bioconda.github.io/recipes/ashlar/README.html?highlight=ashlar#package-package%20&#x27;ashlar&#x27;
but when I try the container pull command on the page I get:
```
(ashlar-recipe) pollen@system76-pc:/tmp$ docker pull quay.io/biocontainers/ashlar:latest
Error response from daemon: unauthorized: access to the requested resource is not authorized
```
Shall I go ahead and request that it be made public as suggested?